UserAdminBundle/EventSubscriber/ResetPasswordSubscriber.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
@alavieille : github is sending an alert on the first char, are you sure of your filde encoding ?